src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/header.html.twig

```diff
@@ -1,21 +1,6 @@
 <div id="header">
     <h1><a href="{{ path('_profiler_home') }}">{{ source('@WebProfiler/Icon/symfony.svg') }} Symfony Profiler</a></h1>
 
-    <div class="sf-tabs" data-processed="true">
```

Comment:
We may add more types in the future, hopefully consumed messages in 7.1.
There may be more space here to do so than in the search panel.

Reply:
I see. We could quickly move the toggle to show it above search results:

![profile-type-toggle](https://github.com/symfony/symfony/assets/73419/d4d98db7-ba2a-417f-8851-37141ee4c9bd)

Do you prefer to do that now or in the near future when we add more profile types?

Reply:
I would keep it in the search bar for now. If we add Messenger, we can still think about a better solution

Reply:
I'd rather put it in a place that won't change to keep the UI consistent in future versions.

Reply:
Agreed, but it should not be located in the content area. We can put it where proposed by Javier and find another kind of display if needed

Reply:
I've moved the toggle to the top of the search results page:

![search-by-profile-type](https://github.com/symfony/symfony/assets/73419/6a2bb400-0c3c-48ce-a32b-93727bd30c5e)